fpdf/fpdf.py

```diff
@@ -1038,6 +1047,27 @@ def set_line_width(self, width):
         if self.page > 0:
             self._out(f"{width * self.k:.2f} w")
 
+    def set_page_background(self, background):
+        """
+        Sets a background color or image to be drawn every time add_page() is called.
```

Comment:
```suggestion
        Sets a background color or image to be drawn every time `FPDF.add_page()` is called.
```
This will create an hyperlink in the HTML documentation generated by `pdoc`